spec/ameba/rule/lint/unused_local_variable_access_spec.cr

```diff
@@ -0,0 +1,69 @@
+require "../../../spec_helper"
+
+module Ameba::Rule::Lint
+  describe UnusedLocalVariableAccess do
+    subject = UnusedLocalVariableAccess.new
+
+    it "passes if local variables are used in assign" do
+      expect_no_issues subject, <<-CRYSTAL
+        a = 1
```

Comment:
Use `foo`, `bar`, ... naming convention to make it clear that errors cover whole name and not just the first character.

ditto elsewhere